.github/workflows/release.yaml

```diff
@@ -41,6 +44,8 @@ jobs:
 
       - name: Run chart-releaser
         id: chart-releaser
-        uses: helm/chart-releaser-action@v1.2.0
+        uses: helm/chart-releaser-action@v1
+        with:
+          skip_existing: false
```

Comment:
@matthewelwell Can we add a comment explaining the reason?

Reply:
Yep, good call - done.